src/runtime/components/InputNumber.vue

```diff
@@ -56,12 +56,17 @@ export interface InputNumberProps extends Pick<NumberFieldRootProps, 'modelValue
   locale?: string
   class?: any
   ui?: InputNumber['slots']
+  /** Disable the increment button. */
+  disabledIncrement?: boolean
```

Comment:
I think it would be better called `incrementDisabled` and `decrementDisabled` and moved under `incrementIcon` / `decrementIcon`.